### DISCUSSION TITLE
Weird pixels

### DISCUSSION THREAD

**Discussion by MRNobady0** [atom/atom#23144] [Q&A]
I have some weird pixels appearing in the screen, looks like my screen is broken but it isn't. any solutions or reasons?
I did upgrade to Windows 11 recently so that could be an issue?
![Schermafbeelding 2021-10-25 115909](https://user-images.githubusercontent.com/93125794/138675581-ac7e4b5f-8a11-47e2-8582-58f1654ead35.png)

**Comment by Ceocanna** [comment]
You should run a diagnostic on your pc then reboot it, to see if that helps

**Comment by icecream17** [comment]
https://github.com/atom/atom/discussions/22402

**Reply by icecream17** [reply]
> Oh, I found something: https://www.reddit.com/r/vscode/comments/ep1t8d/what_is_going_on_here_accidentally_creating_a/fegknjg?utm_source=share&utm_medium=web2x&context=3
> 
> > Can you turn off hardware acceleration per program? I haven't run graphics in a few years, but that could be a combination of anti-aliased fonts in VS being cyclically anti-aliased by a graphics card?
> > Shooting in the dark here, someone tell me why I'm an idiot.
> 
> Maybe try this
>
> *Originally posted by icecream17 in https://github.com/atom/atom/discussions/22402#discussioncomment-832134*

**Reply by MRNobady0** [reply]
Couldn't find anything about disabling Hardware acceleration, but i did troubleshooting and then use settings for Windows 8 wich seems to work
